### PR TITLE
ci: update supported versions

### PR DESCRIPTION
**Pull request by kortschak**
Please take a look.

### CODE REVIEW

**Comment by kortschak** [comment]
AppVeyor is slow.

File: appveyor.yml [hunk 1]
```diff
@@ -13,9 +13,9 @@ environment:
   # Do not move these lines; they are referred to by README.md.
   # Versions of go that are explicitly supported by gonum.
   matrix:
-    - GOROOT: 'c:\go19'
     - GOROOT: 'c:\go110'
     - GOROOT: 'c:\go111'
+    - GOROOT: 'c:\go112'
```
Comment (sbinet) [inline appveyor.yml]: shouldn't we follow the same convention than in travis and put 1.12 first?

Comment (kortschak) [reply appveyor.yml]: We could, but the appveyor tests are pretty short. Another PR.

**Comment by kortschak** [comment]
I'm not particularly fussed about AppVeyor failures here. They'll go away soon.

**Pull request merged**